heka/meta/heka.yml

```diff
@@ -336,7 +336,8 @@ aggregator:
           threshold: 50
           function: percent
     - status: okay
-    # A policy which is typically used for stateless clusters
+    # A policy that is used to derive a cluster status based
+    # on the status okay or down status for its members.
```

Comment:
So either the purpose of the `availability_of_members` policy should be changed (along with its description) or the mysql alarm cluster definition should be changed. I don't know right now.